README.md

```diff
@@ -6,24 +6,40 @@ RNA-seq reporting workflow designed to post-process, summarise and visualise an
 ## Table of contents
 
 <!-- vim-markdown-toc GFM -->
-* [Installation](#installation)
-* [Docker](#docker)
-* [Workflow](#workflow)
-* [Reference data](#reference-data)
-    * [External reference cohorts](#external-reference-cohorts)
-    * [Internal reference cohort](#internal-reference-cohort)
-* [Input data](#input-data)
-  * [WTS](#wts)
-     * [bcbio-nextgen](#bcbio-nextgen)
-     * [Dragen RNA](#dragen-rna)
-  * [WGS](#wgs)
-* [Usage](#usage)
-  * [Arguments](#arguments)
-  * [Examples](#examples)
-  	 * [WTS data only](#1-wts-data-only)
-  	 * [WTS and WGS data](#2-wts-and-wgs-data)
-  	 * [WTS WGS and clinical data](#3-wts-wgs-and-clinical-data)
-  * [Output](#output)
+- [RNAsum](#rnasum)
+  - [Table of contents](#table-of-contents)
+  - [Installation](#installation)
+  - [Docker](#docker)
+  - [Workflow](#workflow)
+  - [Reference data](#reference-data)
+    - [External reference cohorts](#external-reference-cohorts)
+          - [Note](#note)
+    - [Internal reference cohort](#internal-reference-cohort)
+          - [Note](#note-1)
+  - [Input data](#input-data)
+    - [WTS](#wts)
+      - [bcbio-nextgen](#bcbio-nextgen)
+          - [Note](#note-2)
+      - [Dragen RNA](#dragen-rna)
+    - [WGS](#wgs)
+  - [Usage](#usage)
+    - [Arguments](#arguments)
+          - [Note](#note-3)
+    - [Run via AWS-Batch](#run-via-aws-batch)
+    - [Examples](#examples)
+          - [Note](#note-4)
+      - [1. WTS data only](#1-wts-data-only)
+        - [bcbio-nextgen](#bcbio-nextgen-1)
+        - [Dragen RNA](#dragen-rna-1)
+      - [2. WTS and WGS data](#2-wts-and-wgs-data)
+        - [bcbio-nextgen](#bcbio-nextgen-2)
+        - [Dragen RNA](#dragen-rna-2)
+      - [3. WTS WGS and clinical data](#3-wts-wgs-and-clinical-data)
+        - [bcbio-nextgen](#bcbio-nextgen-3)
+        - [Dragen RNA](#dragen-rna-3)
+    - [Output](#output)
+      - [Report](#report)
```

Comment:
I'd skip that part in the TOC...

Reply:
And why is so?

Reply:
The "Results" paragraph is really short, just one sentence. But I'm happy to keep it in the TOC if you feel that's relevant